src/views/Leaderboard.vue

```diff
@@ -0,0 +1,202 @@
+<style>
+table {
+  font-family: arial, sans-serif;
+  border-collapse: collapse;
+  width: 100%;
+}
+td,
+th {
+  border: 1px solid #dddddd;
+  text-align: left;
+  padding: 8px;
+}
+tr:nth-child(even) {
+  background-color: #dddddd;
+}
+</style>
+
+<template>
+  <div class="home">
+    <div
+      style="
+        width: 100%;
+        display: flex;
+        justify-content: flex-end;
+        padding-bottom: 10px;
+      "
+    >
+      <button class="btn btn-primary" @click="onFirst" style="margin: 5px">
+        First
+      </button>
+      <button class="btn btn-primary" @click="onPrevious" style="margin: 5px">
+        {{ "<" }}
+      </button>
+      <button
+        class="btn btn-primary"
+        @click="onSwitchElementsToDisplay"
+        style="margin: 5px"
+      >
+        {{ `${currentIndex + 1} - ${currentIndex + elementsToDisplay}` }}
+      </button>
+      <button class="btn btn-primary" @click="onNext" style="margin: 5px">
+        {{ ">" }}
+      </button>
+      <button class="btn btn-primary" @click="onLast" style="margin: 5px">
+        Last
+      </button>
+    </div>
+    <table style="width: 100%">
+      <tr>
+        <th>Rank</th>
+        <th>Address</th>
+        <th>Quantity</th>
+        <th>Percentage</th>
+      </tr>
+      <tr v-for="(rank, index) in elementsToDisplay" :key="index">
+        <td>{{ rank + currentIndex }}</td>
+        <td>{{ leaderboard[index + currentIndex].address }}</td>
+        <td>{{ leaderboard[index + currentIndex].token.length }}</td>
+        <td>
+          {{
+            (
+              (leaderboard[index + currentIndex].token.length / 26969) *
+              100
+            ).toFixed(2)
+          }}%
+        </td>
+      </tr>
+    </table>
+    <div
+      style="
+        width: 100%;
+        display: flex;
+        justify-content: flex-end;
+        padding-top: 10px;
+      "
+    >
+      <button class="btn btn-primary" @click="onFirst" style="margin: 5px">
+        First
+      </button>
+      <button class="btn btn-primary" @click="onPrevious" style="margin: 5px">
+        {{ "<" }}
+      </button>
+      <button
+        class="btn btn-primary"
+        @click="onSwitchElementsToDisplay"
+        style="margin: 5px;"
+      >
+        {{ `${currentIndex + 1} - ${currentIndex + elementsToDisplay}` }}
+      </button>
+      <button class="btn btn-primary" @click="onNext" style="margin: 5px">
+        {{ ">" }}
+      </button>
+      <button class="btn btn-primary" @click="onLast" style="margin: 5px">
+        Last
+      </button>
+    </div>
+  </div>
+</template>
+
+<script>
+import holder from "../store/data/holder.json";
```

Comment:
that should be rather a blockchain information than a static data set

Reply:
Yes, I know. But requesting all that data takes some time, at least the way I do it, so I created this "snapshot" to boost start everything. Is there a better way to request the blockchain data? A faster one?